apps/web/package.json

```diff
@@ -61,8 +61,9 @@
     "@monaco-editor/react": "^4.6.0",
     "@novu/design-system": "^0.24.2",
     "@novu/notification-center": "^0.24.2",
-    "@novu/shared": "^0.24.2",
-    "@novu/shared-web": "^0.24.2",
+    "@novu/novui": "^0.0.1",
+    "@novu/shared": "^0.24.1",
+    "@novu/shared-web": "^0.24.1",
```

Comment:
Is this revert intentional?

Reply:
Good catch. I'd say it comes from a rebase/merge conflict.